### PR TITLE
Add supermarket ctl command

### PR DESCRIPTION
**Pull request by nellshamrell**
Adds in a command to make an existing Supermarket user an admin. 

User would run:
```bash
  $ supermarket-ctl make_admin supermarket_username
```
This will trigger a rake task in the Supermarket repo which will add the "admin" role to the user.

This should be released in tandem with https://github.com/chef/supermarket/pull/1054

### CODE REVIEW

**Comment by smith** [comment]
If you look at all of the [chef-server-ctl](https://docs.chef.io/ctl_chef_server.html) commands, as well as the built-in `service-list` and `show-config`, they use hyphens instead of underscores. We should do the same as well.

**Comment by nellshamrell** [comment]
Done!

File: cookbooks/omnibus-supermarket/files/default/ctl-commands/make-admin.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+require 'mixlib/shellout'
+
+# supermarket-ctl make_admin username
+add_command 'make-admin', 'Make a Supermarket user an admin', 2 do
+
+  # Find username arg
+  username = ARGV[3]
+
+  # Run rake task
+  command_text = "cd /opt/supermarket/embedded/service/supermarket && sudo RAILS_ENV=\"production\" env PATH=/opt/supermarket/embedded/bin bin/rake user:make_admin user=\"#{username}\""
```
Comment (smith) [inline cookbooks/omnibus-supermarket/files/default/ctl-commands/make-admin.rb]: Why is it `sudo RAILS_ENV=\"production\" env PATH=/opt/...`? Wouldn't `sudo env RAILS_ENV=\"production\" PATH=/opt/...` make more sense? I suppose if it works it's fine but it looks strange.

**Comment by smith** [comment]
:+1:

**Pull request merged**